scripts/docker/buildpack-deps/Dockerfile.emscripten

```
@@ -32,11 +32,14 @@
 # apparently this currently breaks due to conflicting compatibility headers.
 # Using $(em-config CACHE)/sysroot/usr seems to work, though, and still has cmake find the
 # dependencies automatically.
-FROM emscripten/emsdk:2.0.33 AS base
-LABEL version="11"
+FROM emscripten/emsdk:3.1.19 AS base
+LABEL version="12"
 
 ADD emscripten.jam /usr/src
 RUN set -ex; \
+\
+	apt-get update && apt-get install lz4; \
+\
```

Docker already warned about the "empty continuation" lines, so I fixed that as well while I was already touching the file..

---

I'd add an indent here:
```suggestion
    \
```

Also, all these lines should really end with `&&` so that any error makes the whole set fail. Otherwise the return code depends only on the last command, which is `rm -r /usr/src/boost_1_75_0` here :)